Wikipedia/Code/WMFLoginViewController.swift

```diff
@@ -319,4 +324,8 @@ class WMFLoginViewController: WMFScrollViewController, UITextFieldDelegate, WMFC
         scrollContainer.backgroundColor = theme.colors.paperBackground
         captchaViewController?.apply(theme: theme)
     }
+    
+    func setViewControllerUserInteraction(enabled: Bool) {
+        self.view.isUserInteractionEnabled = enabled
```

Comment:
This should dim the view to indicate that user interaction is disabled. Can be addressed in a follow on PR.